styles/colors-gallium-day.less

```diff
@@ -146,7 +148,7 @@
 @agate-keypad-button-focus-bg-color:        @agate-accent;
```

Comment:
I think if we update @agate-checkbox-focus-bg-color, it seems that @agate-keypad-button-focus-bg-color needs to be updated too. Could you please check it?